### DISCUSSION TITLE
add close on click outside of drawer molecule

### DISCUSSION THREAD

**Discussion by PablitoGS** [SUI-Components/sui-components#1644] [Ideas and new features]
(https://sui-components.vercel.app/workbench/molecule/drawer/demo) 
add close on click outside of component ?

**Comment by turolopezsanabria** [comment]
@andresin87 has comments :)

**Reply by andresin87** [reply]
We could give 2 different trigger behaviors like in tooltip or also [popover](https://sui-components.vercel.app/workbench/atom/popover/demo):
Given 2 diferent elements:
- target: the element which acts over the drawer.
- targeted: the drawer element

There are 2 or 3 different behaviors under the 'trigger' prop:
- manual(default): controlled open-close behavior.
- focus: 
  - opens by clicking on an target element
  - closes clicking on any element which is not the targeted (drawer)

(- legacy:
  - opens by clicking on an target element
  - closes clicking on any element which is not the target or targeted)
